tests/nativeapp/codegen/snowpark/__snapshots__/test_extension_function_utils.ambr

```diff
@@ -1,33 +1,97 @@
 # serializer version: 1
-# name: test_enrich_ex_fn
-  "'a/b/c.py','a/b/c','a/b/c.py','a/b/c.jar','a/b/c'"
+# name: test_deannotate_preserves_specified_annotations
+  '''
+  import snowflake.snowpark
+  
+  from typing import Optional
+  from snowflake.snowpark.session import Session
+  from snowflake.snowpark.functions import col, sum, sproc
+  
+  
+  @custom
+  #: @sproc(native_app_params={'schema': 'math', 'application_roles': ['app_public', 'app_admin']})
```

Comment:
In case a user has a linter installed like ours where the annotation can be spread out over multiple lines, will this include a multi-line comment?

Reply:
Right! Added this case, it will work as expected, thanks for the suggestion.